src/lib/config.ts

```diff
@@ -110,6 +110,7 @@ class default_config {
      * They consist of key sequences mapped to ex commands.
      */
     inputmaps = {
+        "<S-Insert>": "mode normal",
```

Comment:
I'm really confused. Why would shift insert put you in normal mode from insert / input?

I don't think this PR should be making any changes to the config at all.

Reply:
>Why would shift insert put you in normal mode from insert / input?

I don't know. I don't remember if it was already there or if I added it accidentally when trying to re-add the binding for ignoremaps. I nuked every change to config.ts so everything should be alright now.